### PR TITLE
Add NT field to schema so that field can be parsed out of VCFs

### LINKED ISSUE #471: NT column in info field is not parsed out
`INFO` field looks like 
```
QSS_ref NT=hom;QSS=55;QSS_NT=55;SGT=GG->GT;SOMATIC;TQSS=1;TQSS_NT=1
```

NT is not showing as a column, but QSS and on are.

### PR DESCRIPTION
**Pull request by ihodes**
fixes #471

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/cycledash/476)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -61,6 +61,7 @@ CREATE TABLE genotypes (
        "info:MQ" TEXT, -- RMS mapping quality
        "info:MQ0" TEXT, -- [INTEGER] number of mapQ == 0 reads covering this record
        "info:NS" TEXT, -- [INTEGER] number of samples with data
+       "info:NT" TEXT, -- [INTEGER] number of samples with data
```
Comment (arahuja) [inline schema.sql]: comment line seems wrong?

**Comment by danvk** [comment]
Do you need to update the dpxdt database dump?

**Comment by ihodes** [comment]
@arahuja & @danvk  good catch, will change

**Comment by ihodes** [comment]
@danvk back to you; don't need to edit the dpxdt data dump it turns out.

**Pull request merged**